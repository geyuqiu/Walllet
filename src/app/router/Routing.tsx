import React from "react";
import {Route, Switch} from "react-router-dom";

import {FeeInput} from "../pages/FeeInput/FeeInput";
import {WalletView} from "../pages/Wallet/WalletView";

export const Routing = () => (
	<Switch>
		<Route path="/fee">
			<WalletView />
		</Route>
		<Route path="/">
			<FeeInput />
		</Route>
	</Switch>
);
