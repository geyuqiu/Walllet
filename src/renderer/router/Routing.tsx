import React from "react";
import { Route, Switch } from "react-router-dom";

import { FeeInput } from "../components/FeeInput/FeeInput";
import { Wallet } from "../components/Wallet/Wallet";

export const Routing = () => (
	<Switch>
		<Route path="/fee">
			<FeeInput />
		</Route>
		<Route path="/">
			<Wallet />
		</Route>
	</Switch>
);
