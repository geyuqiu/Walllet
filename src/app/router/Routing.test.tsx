import { render } from "@testing-library/react";
import React from "react";
import { MemoryRouter } from "react-router-dom";

import { Routing } from "./Routing";

describe("Routing", () => {
	it("should match snapshot", () => {
		const { container } = render(
			<MemoryRouter>
				<Routing />
			</MemoryRouter>
		);
		expect(container).toMatchSnapshot();
	});
});
