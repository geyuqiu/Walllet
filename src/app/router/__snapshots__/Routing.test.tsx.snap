// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routing should match snapshot 1`] = `
<div>
  <div
    class="Cardstyles__Wrapper-xnuhyv-0 iAsJMU bg-green-dark sm:flex sm:justify-center"
    data-testid="Card"
  >
    <div
      class="Cardstyles__Wrapper-xnuhyv-0 iAsJMU bg-black-darkest text-white rounded-lg w-full max-w-screen-xl flex justify-center lg:items-center flex-col lg:flex-row"
      data-testid="Card"
    >
      <div
        class="flex items-center my-auto cursor-pointer"
      >
        <div
          class="sc-gsTCUz dXshFa"
          width="57"
        >
          <svg
            width="44"
          >
            logo.svg
          </svg>
        </div>
        <span
          class="text-2xl font-bold lg:hidden xl:block"
          data-testid="logo__text"
        >
          ARK Wallet
        </span>
      </div>
      <div
        class="lg:hidden"
      >
        <div
          class="Divider-sc-10xryrv-0 ijsPwK border-black-light dark:border-theme-secondary-600"
          type="horizontal"
        />
      </div>
      <div
        class="lg:contents hidden"
      >
        <div
          class="Divider-sc-10xryrv-0 dCquGe border-black-light dark:border-theme-secondary-600"
          type="vertical"
        />
      </div>
      <div
        class="relative border border-gray-darkest border-opacity-10 rounded-l-3xl rounded-r-3xl"
      >
        <span
          data-testid="dropdown__toggle"
        >
          <button
            class="flex justify-between items-center h-11 w-full"
          >
            <div
              class="flex items-center"
            >
              <div
                class="hidden sm:block"
              >
                <div
                  class="Circle__CircleWrapper-sc-1hq8kv9-0 cqSuMy rounded-l-3xl border-r bg-black-light ring-theme-background"
                >
                  <div
                    class="sc-bdfBwQ dfWOLJ ml-2"
                    data-testid="dropdown__left__icon"
                    height="15"
                    width="15"
                  >
                    <svg
                      fill="#000000"
                      stroke="#FBC457"
                    >
                      address.svg
                    </svg>
                  </div>
                </div>
              </div>
              <div
                class="ml-5"
              >
                <span
                  class="hidden sm:block lg:hidden xl:block"
                >
                  AdzbhuDTyhnfAqepZzVcVsgd1Ym6FgETuW
                </span>
                <span
                  class="block sm:hidden"
                >
                  AdzbhuD...m6FgETuW
                </span>
                <span
                  class="hidden lg:block xl:hidden"
                >
                  AdzbhuDTyhn...1Ym6FgETuW
                </span>
              </div>
            </div>
            <div
              class="sc-bdfBwQ dfWOLJ mx-5"
              data-testid="dropdown__right_icon"
              height="15"
              width="15"
            >
              <svg>
                chevron-down.svg
              </svg>
            </div>
          </button>
        </span>
      </div>
      <div
        class="hidden lg:contents"
      >
        <div
          class="Divider-sc-10xryrv-0 dCquGe border-black-light dark:border-theme-secondary-600"
          type="vertical"
        />
      </div>
      <div
        class="flex flex-row mt-6 lg:mt-0"
      >
        <div
          class="hidden sm:block"
        >
          <div
            class="Circle__CircleWrapper-sc-1hq8kv9-0 cqSuMy border rounded-full mr-3 ring-theme-background"
          >
            <div
              class="sc-bdfBwQ dfWOLJ"
              height="15"
              width="15"
            >
              <svg
                fill="#000000"
                stroke="#FBC457"
              >
                balance.svg
              </svg>
            </div>
          </div>
        </div>
        <div
          class="flex flex-col"
        >
          <span
            class="text-gray-darkest"
          >
            Balance
          </span>
          <span
            class="font-bold"
            data-testid="balance__Amount"
          >
            10,043.109 DARK
          </span>
        </div>
      </div>
    </div>
  </div>
  <section
    class="ml-6 mr-3 sm:mx-8 sm:flex sm:justify-center pt-12"
  >
    <p>
      Loading!
    </p>
  </section>
</div>
`;
